recipe/meta.yaml

```
@@ -8,11 +8,11 @@ package:
   version: {{ version }}
 
 source:
-  url: https://files.pythonhosted.org/packages/8c/40/096ceb30b86ea829031662dfcd1d3014a7e1634aff2bc72f3680c97e084c/zarr-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
```

```suggestion
  sha256: 033859c5603dc9c29e53af494ede24b42f1b761d2bb625466990a3b8a9afb792
```

There is also virtually no reason not to copy the `sha256` in there it is a variable that never gets reused.

---

I've decided to leave this as is so that we can keep most updates to only the first few lines.